README.rst

```diff
@@ -8,7 +8,10 @@ Use and Installation:
 ---------------------
 
 * pip install via github url
-* Add ``eultheme`` and ``widget_tweaks`` to **INSTALLED_APPS**
+* Add ``eultheme``, ``widget_tweaks``, and ``downtime`` to **INSTALLED_APPS**
+* Add ``eultheme.middleware.DownpageMiddleware`` to **MIDDLEWARE_CLASSES** after
```

Comment:
Occurs to me, perhaps we should add sample code for this so it is easier to cut & paste into settings?